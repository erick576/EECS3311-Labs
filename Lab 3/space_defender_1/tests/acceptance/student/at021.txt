play(5, 10, 5, 1)
move(C, 5)
move(C, 10)
fire
fire
pass
move(C, 5)
move(C, 2)
fire
move(E, 4)
move(C, 5)
play(5, 10, 5, 1)
fire
move(D, 2)
move(C, 4)
pass
pass
