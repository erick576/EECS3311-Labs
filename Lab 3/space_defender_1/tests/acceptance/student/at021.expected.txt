  Welcome to Space Defender Version 1.
->play(5,10,5,1)
  state:1.0, ok
      1  2  3  4  5  6  7  8  9 10
    A _  _  _  _  _  _  _  _  _  _
    B _  _  _  _  _  _  _  _  _  _
    C S  _  _  _  _  _  _  _  _  _
    D _  _  _  _  _  _  _  _  _  _
    E _  _  _  _  _  _  _  _  _  _
->move(C,5)
  state:2.0, ok
  The Starfighter moves: [C,1] -> [C,5]
      1  2  3  4  5  6  7  8  9 10
    A _  _  _  _  _  _  _  _  _  _
    B _  _  _  _  _  _  _  _  _  _
    C _  _  _  _  S  _  _  _  _  _
    D _  _  _  _  _  _  _  _  _  _
    E _  _  _  _  _  _  _  _  _  _
->move(C,10)
  state:3.0, ok
  The Starfighter moves: [C,5] -> [C,10]
      1  2  3  4  5  6  7  8  9 10
    A _  _  _  _  _  _  _  _  _  _
    B _  _  _  _  _  _  _  _  _  _
    C _  _  _  _  _  _  _  _  _  S
    D _  _  _  _  _  _  _  _  _  _
    E _  _  _  _  _  _  _  _  _  _
->fire
  state:4.0, ok
  The Starfighter fires a projectile at: [C,10]
      1  2  3  4  5  6  7  8  9 10
    A _  _  _  _  _  _  _  _  _  _
    B _  _  _  _  _  _  _  _  _  _
    C _  _  _  _  _  _  _  _  _  S
    D _  _  _  _  _  _  _  _  _  _
    E _  _  _  _  _  _  _  _  _  _
->fire
  state:5.0, ok
  The Starfighter fires a projectile at: [C,10]
      1  2  3  4  5  6  7  8  9 10
    A _  _  _  _  _  _  _  _  _  _
    B _  _  _  _  _  _  _  _  _  _
    C _  _  _  _  _  _  _  _  _  S
    D _  _  _  _  _  _  _  _  _  _
    E _  _  _  _  _  _  _  _  _  _
->pass
  state:6.0, ok
  The Starfighter stays at: [C,10]
      1  2  3  4  5  6  7  8  9 10
    A _  _  _  _  _  _  _  _  _  _
    B _  _  _  _  _  _  _  _  _  _
    C _  _  _  _  _  _  _  _  _  S
    D _  _  _  _  _  _  _  _  _  _
    E _  _  _  _  _  _  _  _  _  _
->move(C,5)
  state:7.0, ok
  The Starfighter moves: [C,10] -> [C,5]
      1  2  3  4  5  6  7  8  9 10
    A _  _  _  _  _  _  _  _  _  _
    B _  _  _  _  _  _  _  _  _  _
    C _  _  _  _  S  _  _  _  _  _
    D _  _  _  _  _  _  _  _  _  _
    E _  _  _  _  _  _  _  _  _  _
->move(C,2)
  state:8.0, ok
  The Starfighter moves: [C,5] -> [C,2]
      1  2  3  4  5  6  7  8  9 10
    A _  _  _  _  _  _  _  _  _  _
    B _  _  _  _  _  _  _  _  _  _
    C _  S  _  _  _  _  _  _  _  _
    D _  _  _  _  _  _  _  _  _  _
    E _  _  _  _  _  _  _  _  _  _
->fire
  state:9.0, ok
  The Starfighter fires a projectile at: [C,2]
      1  2  3  4  5  6  7  8  9 10
    A _  _  _  _  _  _  _  _  _  _
    B _  _  _  _  _  _  _  _  _  _
    C _  S  *  _  _  _  _  _  _  _
    D _  _  _  _  _  _  _  _  _  _
    E _  _  _  _  _  _  _  _  _  _
->move(E,4)
  state:10.0, ok
  A projectile moves: [C,3] -> [C,4]
  The Starfighter moves: [C,2] -> [E,4]
      1  2  3  4  5  6  7  8  9 10
    A _  _  _  _  _  _  _  _  _  _
    B _  _  _  _  _  _  _  _  _  _
    C _  _  _  *  _  _  _  _  _  _
    D _  _  _  _  _  _  _  _  _  _
    E _  _  _  S  _  _  _  _  _  _
->move(C,5)
  state:11.0, ok
  A projectile moves: [C,4] -> [C,5]
  The Starfighter moves and collides with a projectile: [E,4] -> [C,5]
      1  2  3  4  5  6  7  8  9 10
    A _  _  _  _  _  _  _  _  _  _
    B _  _  _  _  _  _  _  _  _  _
    C _  _  _  _  X  _  _  _  _  _
    D _  _  _  _  _  _  _  _  _  _
    E _  _  _  _  _  _  _  _  _  _
  The game is over. Better luck next time!
->play(5,10,5,1)
  state:12.0, ok
      1  2  3  4  5  6  7  8  9 10
    A _  _  _  _  _  _  _  _  _  _
    B _  _  _  _  _  _  _  _  _  _
    C S  _  _  _  _  _  _  _  _  _
    D _  _  _  _  _  _  _  _  _  _
    E _  _  _  _  _  _  _  _  _  _
->fire
  state:13.0, ok
  The Starfighter fires a projectile at: [C,1]
      1  2  3  4  5  6  7  8  9 10
    A _  _  _  _  _  _  _  _  _  _
    B _  _  _  _  _  _  _  _  _  _
    C S  *  _  _  _  _  _  _  _  _
    D _  _  _  _  _  _  _  _  _  _
    E _  _  _  _  _  _  _  _  _  _
->move(D,2)
  state:14.0, ok
  A projectile moves: [C,2] -> [C,3]
  The Starfighter moves: [C,1] -> [D,2]
      1  2  3  4  5  6  7  8  9 10
    A _  _  _  _  _  _  _  _  _  _
    B _  _  _  _  _  _  _  _  _  _
    C _  _  *  _  _  _  _  _  _  _
    D _  S  _  _  _  _  _  _  _  _
    E _  _  _  _  _  _  _  _  _  _
->move(C,4)
  state:15.0, ok
  A projectile moves: [C,3] -> [C,4]
  The Starfighter moves and collides with a projectile: [D,2] -> [C,4]
      1  2  3  4  5  6  7  8  9 10
    A _  _  _  _  _  _  _  _  _  _
    B _  _  _  _  _  _  _  _  _  _
    C _  _  _  X  _  _  _  _  _  _
    D _  _  _  _  _  _  _  _  _  _
    E _  _  _  _  _  _  _  _  _  _
  The game is over. Better luck next time!
->pass
  state:15.1, error
  Not in game.
->pass
  state:15.2, error
  Not in game.
